app/assets/javascripts/app/radio-btn.js

```diff
@@ -0,0 +1,39 @@
+import 'classlist.js';
+
+function clearRadioBtn(name) {
+  const radioGroup = document.querySelectorAll(`input[name='${name}']`);
+
+  for (let i = 0; i < radioGroup.length; i++) {
+    radioGroup[i].parentNode.parentNode.classList.remove('bg-light-blue');
+  }
+}
+
+function radioBtn() {
```

Comment:
I wonder if there is an alternative name for this function that would be clearer. I think `clearRadioBtn` is nicely named, for example. It seems like this function is adding and removing styles for radio buttons. Maybe a name related to that?